Modules/_math.h

```diff
@@ -7,8 +7,9 @@
 static double
 _Py_log1p(double x)
 {
-    /* Some platforms supply a log1p function but don't respect the sign of
-       zero:  log1p(-0.0) gives 0.0 instead of the correct result of -0.0.
+    /* Some platforms (e.g. MacOS X 10.8, see #59682) supply a log1p function
```

Comment:
Python sometimes changes bugtracker systems (SourceForge bugtracker → Roundup → GitHub Issues) invalidating all relative issue numbers on export. So we need to specify a full number.

```suggestion
    /* Some platforms (e.g. MacOS X 10.8, see gh-59682) supply a log1p function
```

Reply:
Side note: Roundup migration was done [in 2008 (Python 2.6)](https://docs.python.org/2.6/whatsnew/2.6.html#new-issue-tracker-roundup), GitHub  migration was done [last year](https://discuss.python.org/t/github-issues-are-now-live/14967). And nobody knows when and why the next migration will take place.

Reply:
Thanks.  I thought that such references were fixed (that's how I did conversion for sympy) in sources too and now there is some canonical way to address issues.